.pylintrc

```diff
@@ -115,7 +115,7 @@ evaluation=10.0 - ((float(5 * error + warning + refactor + convention) / stateme
 [BASIC]
 
 # Good variable names which should always be accepted, separated by a comma
-good-names=i,j,k,ex,Run,_,d,e,v,o,l,x,ts,f
+good-names=i,j,k,ex,Run,_,d,e,v,o,l,x,ts,f,id
```

Comment:
`id` is not ok as it would shadow the globally scoped python `id` function.
```
In [1]: id?
Signature: id(obj, /)
Docstring:
Return the identity of an object.

This is guaranteed to be unique among simultaneously existing objects.
(CPython uses the object's memory address.)
Type:      builtin_function_or_method

In [2]: id('')
Out[2]: 4401957552
```

Reply:
I added the global disable to avoid doing inline disables for all of the models that have an `id` column (read: all of them). Perhaps I should do a top-level disable at the files containing models instead?